app/controllers/api/campaigns_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::CampaignsController < ApplicationController
+	def index
+		@campaigns = Campaign.all
```

Comment:
Please make the indentation correct and dont repeat this via installing a plugin

Reply:
What if the campaign data is huge, do you think this will timeout ?